docs/reference/modules/ROOT/pages/queries.adoc

```diff
@@ -35,18 +45,19 @@ In the most basic case, a Datalog query works by searching for "subgraphs" in
 the database that match the pattern defined by the clauses. The values within
 these subgraphs are then returned according to the list of return variables
 requested in the `:find` vector within the query.
+////
 
 [#structure]
-== Basic Structure
+=== Basic Structure
 
 A query in Crux is performed by calling `crux/q` on a Crux database snapshot with a quoted map and, optionally, additional arguments.
 
 [source,clojure]
 ----
 include::example$test/crux/docs/examples/query_test.clj[tags=anatomy,indent=0]
 ----
-<1> Database value. Usually, the snapshot view comes from calling `crux/db` on a Crux node
-<2> Query map (vector style queries are also supported)
+<1> Database value. Usually the snapshot view comes from calling `crux/db` on a Crux node
+<2> Query map or vector (ex. `[:find ...etc... ]`, as found in other Datalog databases)
```

Comment:
I would argue it's better to avoid mentioning vector style queries completely rather than refer to the possibility more explicitly